packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -214,7 +215,11 @@ class TextMapPropagator {
           spanContext = this._extractDatadogContext(carrier)
           break
         case 'tracecontext':
-          spanContext = this._extractTraceparentContext(carrier)
+          if (carrier.hasOwnProperty(googleCloudTraceContextKey) && carrier.hasOwnProperty(traceKey)) {
```

Comment:
Not sure I understand this part. It seems to be trying to handle both Datadog and W3C extraction in the W3C block.